__tests__/petTrack.test.js

```diff
@@ -0,0 +1,188 @@
+
```

Comment:
The logic itself looks good here and all the tests pass, but the code formatting is really in need of fixing. Some general rules to bear in mind:
- The indentation should be the same throughout the whole project. Most of the code uses 4 spaces so you should stick to that here too
- There should be a clear line between each `it` block to make the file easier to read
- If more than a couple of lines of setup needed in each test its good to have a clear line before the `expect`  assertions start, just to make it easier to read
- You can have your IDE do a lot of this for you by running prettier across the code, saves you a lot of manual work!